src/viadot/sources/__init__.py

```diff
@@ -5,13 +5,15 @@
 from .genesys import Genesys
 from .sharepoint import Sharepoint
 from .trino_source import Trino
+from .sql_server import SQLServer
 
 __all__ = [
     "CloudForCustomers",
     "ExchangeRates",
     "Genesys",
     "Sharepoint",
     "Trino",
+    "SQLServer",
 ]
 
```

Comment:
https://github.com/dyvenia/viadot/blob/f5e087046f26c86453ddb091fc643ca27f6e6d12/src/viadot/sources/__init__.py#L34-L38 I think we need here additional condition here. Because now even if we install here minio packeges, source will be not imported still. Could you test if that works in container ? 
`if find_spec("s3fs") or find_spec("minio"):`